[BUG] - Automatic Releases don't work properly

## Issue Description

The workflow for automatic releases is not working at all, labels are not being created and consequently neither are releases.

## Steps to Reproduce

1. Push to some branch '...'
2. Create a Pull Request to main '...'
3. Merge the Pull Request '...'
4. The release was not created

## Expected Behavior

The current version is expected to be detected based on semantic commit messages, followed by tag creation and delivery of the new artifact.

## Actual Behavior

Neither the label nor the artifact is created

## Possible Solution

Review the Pipeline configuration and rules on repository branches.

The problem is back with this errors:

error: insufficient permission for adding an object to repository database .git/objects
error: composer.json: failed to insert into database
error: unable to index file 'composer.json'
fatal: updating files failed
Error: Process completed with exit code 128

--- a/.github/workflows/semver.yml
+++ b/.github/workflows/semver.yml
@@ -55,6 +55,9 @@ jobs:
           token: ${{ secrets.GIT_TOKEN }}
           fetch-depth: 0
 
+      - name: Set correct permissions
+        run: sudo chown -R $USER:$USER .
+
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:

--- a/composer.json
+++ b/composer.json
@@ -5,7 +5,7 @@
   "time": "2024-06-02",
   "license": "MIT",
   "type": "project",
-  "minimum-stability": "dev",
+  "minimum-stability": "stable",
   "prefer-stable": true,
   "keywords": [
     "PHP",

